<launch>
  <param command="$(find xacro)/xacro $(find garbage_robot_description)/urdf/garbage_robot.xacro" name="robot_description"/>
  <node args="-param robot_description -urdf -model garbage_robot" name="spawn_urdf" pkg="gazebo_ros" type="spawn_model"/>
  <!-- Transformation Configuration ... Setting Up the Relationships Between Coordinate Frames -->
 <!--<node pkg="tf" type="static_transform_publisher" name="world_to_map" args="0 0 0 0 0 0 world map 10"/>
  <node pkg="tf" type="static_transform_publisher" name="map_to_odom" args="0 0 0 0 0 0 map odom 10"/> 
  <node pkg="tf" type="static_transform_publisher" name="odom_to_base_footprint" args="0 0 0 0 0 0 odom base_footprint 10"/>
 <node pkg="tf" type="static_transform_publisher" name="base_link_broadcaster" args="0 0 0.0 0 0 0 base_footprint base_link 10" />-->

 <!-- Gazebo campus environment -->
  <include file="$(find gazebo_ros)/launch/empty_world.launch">
    <arg name="world_name" value= "$(find garbage_robot_description)/world/myworld.world"/>
    <arg name="paused" value="true"/>
    <arg name="use_sim_time" value="true"/>
    <arg name="gui" value="true"/>
    <arg name="headless" value="false"/>
    <arg name="debug" value="false"/>
  </include>
 <include file="$(find garbage_robot_description)/launch/controller.launch"/>
 <!--<node  name="steering_control_node" pkg="garbage_robot_description" type="stearing.py" />-->
 <node  name="steering_control_node" pkg="garbage_robot_description" type="ackerman.py" />
</launch>
